client/esy.json

```diff
@@ -0,0 +1,6 @@
+{
+    "name": "sketch.sh",
+    "dependencies": {
+        "@opam/reason": "*"
```

Comment:
Don't we need `@opam/ocaml-lsp-server` as dependency? How does it work otherwise?